src/main/resources/application.yaml

```diff
@@ -57,7 +57,9 @@ spring:
       client:
         composite-indicator:
           enabled: false
-
+  mvc:
+    pathmatch:
+      matching-strategy: ant_path_matcher
```

Comment:
Is this needed for Spring Boot 2 update? I am probably wrong, but in my understanding this change was introduced only in Spring Boot 3.

Reply:
think you're right, probably i read ahead in the migration guide